spec/namespaces/_core.yaml

```diff
@@ -9,6 +9,7 @@ paths:
       operationId: info.0
       x-operation-group: info
       x-version-added: '1.0'
+      x-tested: true
```

Comment:
We should add `x-tested` to this [list](https://github.com/opensearch-project/opensearch-api-specification/blob/main/DEVELOPER_GUIDE.md#openapi-extensions) too :)